_posts/2023-02-22-puppetmodule.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: post
+title: Make puppet module documentation great again!
+date: 2023-02-22
+github_username: bastelfreak
+twitter_username: BastelsBlog
+---
+
+A long long time ago I can still remember... [Dominic Cleal](https://github.com/domcleal)
+created [puppetmodule.info](https://www.puppetmodule.info/). Inspired by
+[rubydoc.info](https://rubydoc.info/), the site generates documentation for
+Puppet modules with [puppet-strings](). puppetmodule.info was created around
+2015. Since then Dominic moved on and isn't active anymore in the Puppet
+ecosystem and the site died.
+
+Vox Pupuli adopted it by the end of 2022. We updated the source code to work on
+modern Ruby and enabled TLS for the domain.
+
+You want to use puppetmodule.info for your own modules? Just publish to the
+forge! We scrape the forge API on a regular basis to import new releases
+automatically. You can also add a badge to your README.md:
+
+```
+![puppetmodule.info docs](http://www.puppetmodule.info/images/badge.png)](http://www.puppetmodule.info/m/puppet-borg)
+```
+Which will look like this:
+
+[![puppetmodule.info docs](http://www.puppetmodule.info/images/badge.png)](http://www.puppetmodule.info/m/puppet-borg)
+
+You want to participate in the development? Checkout the sourcecode!
+
+https://github.com/voxpupuli/puppetmodule.info
+
+You like Ops work and want to help running the server? Ping us on Slack or IRC (links in the footer)
+
+You love Vox Pupuli? You can now sponsor us! Check https://opencollective.com/vox-pupuli or https://github.com/sponsors/voxpupuli
+
+
+Thanks Hetzner Online for sponsoring the cloud resources to host puppetmodule.info!
+[!Sponsored by Hetzner Online GmbH]({{ site.url }}{{ site.baseurl }}/static/images/hetzner_cloud_logo.svg)](https://www.hetzner.com)
```

Comment:
I'm not sure how big the image will be, but I also failed to test this locally. maybe someone else has a better understanding about jekyll